tenant_schemas_celery/test_app.py

```diff
@@ -12,6 +12,7 @@ class CeleryConfig:
         CELERY_RESULT_BACKEND = 'rpc://'
         CELERY_RESULT_PERSISTENT = False
         CELERY_ALWAYS_EAGER = False
+        CELERY_TASK_TENANT_CACHE_SECONDS = os.environ.get("TASK_TENANT_CACHE_SECONDS", 10)
```

Comment:
Does it have any use to allow it to be parametrized by an env variable? Like, does changing it make tests cover different cases?

Reply:
Not required. Removed